components/SingleArticle.jsx

```diff
@@ -16,6 +18,13 @@ const SingleArticle = ({article}) => {
       setIsLoading(false);
     })
   }, [id])
+
+  useEffect(() => {
```

Comment:
could do this comments fetching and rendering of the comments in a separate component.

Reply:
keeps things tidy